bands/templates/bands/agreement-base.html

```diff
@@ -84,7 +86,9 @@
     </h3>
     <p>
         {% block band_text_parking_inner %}
-        To be included with hotel room for <b>{{ band.vehicles }} vehicle{{ band.vehicles|pluralize }}</b>.
+        To be included
```

Comment:
yea, this is my interpretation based on how we've done it in the past

if there's a different policy later, supermag folks will let us know